wp-content/plugins/wporg-learn/views/locales-list.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace WPOrg_Learn\Locale;
+
+defined( 'WPINC' ) || die();
+
+/** @var array $locales */
+?>
+
+<script type="text/javascript" id="wporg-learn-locales">
+	var wporgLearnLocales = [
```

Comment:
Maybe there is a better way to namespace this?

Reply:
I was thinking this wasn't going to be compiled so was best in es5, but you're right, our browser support policy doesn't require it.